examples/AN00156_i2c_master_example/doc/rst/AN00156.rst

```diff
@@ -224,31 +220,19 @@ on the xCORE device::
 Once this command has executed the application will be running on the
 xCORE-200 eXplorerKIT.
 
-Launching from xTIMEcomposer Studio
-...................................
-
-From xTIMEcomposer Studio use the run mechanism to download code to xCORE device.
-Select the xCORE binary from the ``bin/`` directory, right click and go to Run
-Configurations. Double click on xCORE application to create a new run configuration,
-enable the xSCOPE I/O mode in the dialog box and then
-select Run.
-
-Once this command has executed the application will be running on the
-xCORE-200 eXplorerKIT.
 
 Running the application
-.......................
+=======================
 
 Once the application is started using either of the above methods there should
 be output printed to the console showing the x, y and z axis values and as you
 move the development board these will change.
 
 |newpage|
 
-References
-----------
-
-.. nopoints::
+***************
+Further Reading
+***************
 
```

Comment:
Links PM seem to like: 

* `XMOS` XTC Tools Installation Guide

    https://xmos.com/xtc-install-guide

  * `XMOS` XTC Tools User Guide

    https://www.xmos.com/view/Tools-15-Documentation

  * `XMOS` application build and dependency management system; `xcommon-cmake`

    https://www.xmos.com/file/xcommon-cmake-documentation/?version=latest

Reply:
Changed!